src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Components/ComponentsApi.cs

```diff
@@ -149,6 +150,7 @@ public static class EventCallback
     public static class EventCallbackOfT
     {
         public const string MetadataName = "Microsoft.AspNetCore.Components.EventCallback`1";
+        public const string DisplayName = "Microsoft.AspNetCore.Components.EventCallback<TValue>";
```

Comment:
I'm not hugely worried because it shouldn't change, but we're not depending on the type parameter name (which normally should matter)
From offline discussion, maybe we could do a display string with "`1" instead?

Reply:
Printing with `` `1 `` etc. is supported, but internal, so we can't use it here. Will stick with the type param names for now.